com.unity.ml-agents/Runtime/Sensors/CameraSensor.cs

```diff
@@ -53,6 +54,18 @@ public SensorCompressionType CompressionType
             m_Name = name;
             m_Shape = GenerateShape(width, height, grayscale);
             m_CompressionType = compression;
+
+            if (m_CompressionType != SensorCompressionType.None)
+            {
+                if (m_Grayscale)
+                {
+                    m_CompressionMapping = new int[] { 1, 1, 1 };
```

Comment:
Should these be  {0, 0, 0} and {0, 1, 2}?

Reply:
0 is currently reserved for channels to be dropped and the mapping index start from 1. I'm thinking about changing it to -1 as drop and starting from 0 like you said.

Reply:
Ah, I see, you're using 1-index in the python code. I think 0-indexing is clearer though.

Reply:
changed to 0-index